plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHive4OnDataLake.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive;
+
+import io.trino.plugin.hive.containers.Hive4MinioDataLake;
+import io.trino.plugin.hive.containers.HiveMinioDataLake;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+
+import static org.junit.jupiter.api.Assumptions.abort;
+
+@Execution(ExecutionMode.SAME_THREAD) // TODO Make custom hive4 image to support running queries concurrently
+class TestHive4OnDataLake
+        extends BaseTestHiveOnDataLake
+{
+    @Override
+    protected HiveMinioDataLake hiveMinioDataLake()
+    {
+        return new Hive4MinioDataLake(bucketName);
+    }
+
+    @Override
+    @Test
+    public void testSyncPartitionOnBucketRoot()
+    {
+        abort("https://github.com/trinodb/trino/issues/24453");
+    }
+
+    @Override
+    @Test
+    public void testUnpartitionedTableExternalLocationOnTopOfTheBucket()
+    {
+        abort("https://github.com/trinodb/trino/issues/24453");
```

Comment:
nit: I didnt mean that you have to change the abort message. More like adding a code comment with issue links

Reply:
sure added a code comment with issue link.